docs/getting-started/openshift/index.md

```diff
@@ -0,0 +1,180 @@
+# ArgoCD Agent Hub and Spoke Cluster Setup Documentation Using Operator
+This document outlines the process of setting up ArgoCD Agent in a hub and spoke cluster architecture. This configuration allows a central ArgoCD instance (the hub) to manage applications deployed across multiple remote Kubernetes clusters (the spokes) through the argocd-agent.
+## Prerequisites
+Before proceeding with the setup, ensure you have the following:
+
+A running openshift/kubernetes cluster designated as the hub, with ArgoCD/openshift GitOps Operator installed.
+One or more Kubernetes clusters designated as spokes, ArgoCD/openshift GitOps Operator installed.
+oc configured to access both hub and spoke clusters.
+Operator must be installed in cluster scope mode.
+Apps in any Namespace should be enabled.
```

Comment:
`Apps in any Namespace` feature must be enabled in the Hub cluster.

Reply:
This doesn't seem like a requirement for autonomous mode. I guess it would depend on how the customers would like to use it

Reply:
Should I mention explicitly for managed?

Reply:
It will be required for autonomous mode too, because the principal will replicate Applications in a namespace with the same name as the agent that created them.